### ISSUE TITLE
clearing line data with set

### ISSUE THREAD

**Issue by Zartexo** [louisnw01/lightweight-charts-python#51]
Hi,

for charts we can use the set() method with an empty dataframe to clear the data. I think the same should work with lines:

```
    def set(self, data: pd.DataFrame, name=''):
        """
        Sets the line data.\n
        :param data: If the name parameter is not used, the columns should be named: date/time, value.
        :param name: The column of the DataFrame to use as the line value. When used, the Line will be named after this column.
        """
        if data.empty or data is None:
            self.run_script(f'{self.id}.series.setData([]); {self.id}.name = "{name}"')
            return
...
```

**Comment by louisnw01** [comment]
Great idea, and thanks for providing the code. I will implement this in the next update.

**Comment by louisnw01** [comment]
Fixed in v1.0.15.

**Comment by julianstannard** [comment]
I cant get this to work. - is there somthing i am doing wrong?
I have tried both giving it no arguments and an empty data frame. Both dont clear the line data that is already in there, but when i dont give it any arguments it complains:
"
TypeError: Line.set() missing 1 required positional argument: 'df'
"
using v1.0.17
